server/src/lib/model/db.ts

```diff
@@ -329,7 +330,8 @@ export default class DB {
             FROM votes
             WHERE votes.clip_id = clips.id AND client_id = ?
           )
-        AND (sentences.clips_count <= 10 OR sentences.clips_count IS NULL)
+        AND (sentences.clips_count <= 10 OR sentences.clips_count = 0)
```

Comment:
Weird, good catch! Double checked and `clips_count` definitely defaults to `0`… 😬

Is there a performance trick going on here that I don't know about, or is this the same as:

```sql
AND (sentences.clips_count <= 10)
```

?

Reply:
GOOD QUESTION, I think this is just a case of my brain skipping the track. Will simplify.